### PR TITLE
Update main.yml

### PR DESCRIPTION
**Pull request by TheGemDev**
@arthuro555 This should work?

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -6,7 +6,10 @@ name: Auto Build
 # events but only for the master branch
 on:
   push:
-    branches: [ master ]
+    branches:
+      - master
+  schedule:
+    - cron: "0 2 * * 1-5"
```
Comment (arthuro555) [inline .github/workflows/main.yml:12]: We don't have unlimited build hours, lets keep on just building when push on master.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -19,8 +22,8 @@ jobs:
     steps:
     
     # Set Up Node
-    - name: Setup Node.js
-      uses: actions/setup-node@v1.4.2
+    - name: Setup Node.js environment
```
Comment (arthuro555) [inline .github/workflows/main.yml:25]: We don't need that action at all as it turns out that node is already or installed. We would only need it if we needed a specific node version, which we don't.

**Review by arthuro555 — COMMENTED** [review]
Sorry I just now saw your pr, I made one as well :/

File: .github/workflows/main.yml (hunk 2 above)
Comment (TheGemDev) [reply .github/workflows/main.yml:25]: Okay

**Comment by TheGemDev** [comment]
Fix l#183

**Pull request closed without merging**